tests/behat/features/file-download.d7.feature

```diff
@@ -0,0 +1,36 @@
+@d7
+Feature: Check that FileDownloadTrait for D7 works
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+    And article content:
+      | title                | field_file                                                   |
+      | [TEST] document page | file;example_files/example_document.txt;example_document.txt |
+    And article content:
+      | title           | field_file                                             |
+      | [TEST] zip page | file;example_files/example_files.zip;example_files.zip |
+
+  @api @download
+  Scenario: Assert that a file can be downloaded
```

Comment:
Please use a step definition in scenario name for clarity

`Scenario: Assert "Then I download file from :url"`